### ISSUE TITLE
Problem with empty fields with no spaces inbetween

### ISSUE THREAD

**Issue by kaofelix** [emacs-vs/rainbow-csv#6]
Hey, first of all, many thanks for this package! I've been jumping to vs-code sometimes to read CSV and this mostly eliminates the need to do so :)

However, I found an issue: 

When there are empty values like this
```
col_a,col_b,col_c,col_d,col_e,col_f,col_g
some value,,,,,another, and another
```
after the first empty field, the syntax highlight doesn't work anymore:

![CleanShot 2024-05-16 at 18 46 05](https://github.com/emacs-vs/rainbow-csv/assets/223625/8da67ccb-5142-4641-aff9-4017bffb512d)

it works fine however, when there are spaces in-between the empty values:

![CleanShot 2024-05-16 at 18 46 19](https://github.com/emacs-vs/rainbow-csv/assets/223625/483df6ac-c037-4228-a0df-767b66a530d4)

For what is worth, this works fine in vs-code.

**Comment by jcs090218** [comment]
Thanks for the bug report! I'll look into it when I have time!

**Comment by jcs090218** [comment]
Fixed in 2a91a23abc05f26d513a1eb59c3e8cf1f42a0336.
